server/api/index.js

```diff
@@ -13,10 +13,16 @@ var version = require( '../../package.json' ).version,
 	cors = require( 'cors' ),
 	oauth = require( './oauth' );
 
-var corsOptions = {
-	origin: 'http://teach-cms-enabled.herokuapp.com',
+var whitelist = config( 'whitelist' );
```

Comment:
can you add documentation at least to the `config/README.md` file that tells us to set this whitelist?